funcx_endpoint/funcx_endpoint/executors/high_throughput/interchange.py

```diff
@@ -142,19 +142,22 @@ def __init__(self,
         suppress_failure : Bool
              When set to True, the interchange will attempt to suppress failures. Default: False
         """
-        self.logdir = logdir
-        try:
-            os.makedirs(self.logdir)
-        except FileExistsError:
-            pass
 
-        start_file_logger("{}/interchange.log".format(self.logdir),
+        if config.interchange_file_logger:
+            logpath = "{}/interchange.log".format(self.logdir)
```

Comment:
Missed this. This does not work on local endpoint.

should be `logdir`, but not `self.logdir`